app/layout.tsx

```diff
@@ -0,0 +1,60 @@
+import { Footer } from 'components/common/Footer';
+import './globals.css';
+import Header from 'components/common/Header';
+import { Metadata } from 'next';
+
+const description = 'Track all transactions on Rango Exchange';
+const socialTitle = 'Rango Exchange Explorer';
+const BASE_URL = 'https://scan.rango.exchange';
+const APP_NAME = 'Rango Exchange Explorer';
+
+export const metadata: Metadata = {
+  title: 'Rango Exchange Explorer',
```

Comment:
Please use constant variables for all metadata, and move them to the `src/constant/meta.ts`.

Reply:
Done in https://github.com/rango-exchange/explorer/pull/32/commits/c35d177529fdc08e919750cd58c88a74602738ec